generator/src/generators/clock.cr

```diff
@@ -0,0 +1,35 @@
+require "../exercise_generator"
+require "../test_case_group"
+
+class ClockGenerator < ExerciseGenerator
+  def exercise_name
+    "clock"
+  end
+
+  def test_cases
+    TestCaseGroup(ClockTestCase).from_json(data).cases[0].cases
+  end
+end
+
+class ClockTestCase < ExerciseTestCase
+  class Input
+    JSON.mapping(
```

Comment:
Anytime I try to generate something I get this error:
```
 16 | JSON.mapping(
           ^------
Error: undefined method 'mapping' for JSON:Module
```

This code is taken from the `acronym` example - perhaps it is the version of Crystal that I have locally?
```
Crystal 1.3.2 [932f193ae] (2022-01-18)

LLVM: 10.0.0
Default target: aarch64-apple-darwin
```